### DISCUSSION TITLE
data reading system for the sensor (Freestyle Libre 2)

### DISCUSSION THREAD

**Discussion by Lloansi** [JohanDegraeve/xdripswift#552] [General]
Hello, my name is Lloansi, and I am a mobile app developer. Together with a group of friends, we are developing an app for glucose monitoring, and we came across this repository.

On our own, we are unable to access the system used by the Freestyle Libre 2, so it would be incredibly helpful if someone could assist us in creating a data reading system for the sensor. We would be very grateful to be able to launch this non-profit app for people with diabetes.

**Comment by paulplant** [comment]
Hi, just saw this discussion now (a bit late, sorry).

I'm not quite sure what it is you're asking. Feel free to reach out if we can help.
